web/src/data/recoil.ts

```diff
@@ -64,6 +64,7 @@ export const selectedCity = atom({
 export const selectedCities = atom<Array<City>>({
   key: "selectedCities",
   default: [],
+  // effects: [localStorageEffect()]
```

Comment:
was this commented by mistake?

Reply:
ya this line itself is not needed will be removed in next push